modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/hooks/useLiferayState.ts

```diff
@@ -12,11 +12,13 @@
  * details.
  */
 
+/// <reference path="../../../../../../../../frontend-js-state-web/src/main/resources/META-INF/resources/State/types.d.ts"/>
```

Comment:
This is obviously the bit that I'd like to make implicit. If we can't get project references to work, we might be able to hack something into `@liferay/npm-scripts` (ie. to inject it in automatically via the `include` property in the `tsconfig.json`).